### ISSUE TITLE
zapier push has not effect

### ISSUE THREAD

**Issue by ecdeveloper** [zapier/zapier-platform-cli#398]
## What would you like to change?
Running `zapier push` seems to be publishing, but when I go check for the changes in the dashboard - I keep seeing the old version. The changes I'm making are, for instance, changing the URL my trigger is sending a request to as part of subscribe hook. I can see in the dashboard that it keeps sending those requests to the old URL, even though I updated the trigger and pushed my update. 

```shell
zapier push --debug
Preparing to build and upload your app.

>> GET https://zapier.com/api/platform/cli/check
<< 200
<< {"success": true}

✔ Copying project to temp directory
⠋ Installing project dependencies

Running npm install --production command in /private/var/folders/b1/ds2mgbv92mzg6gxl7kzvvnmm0000gq/T/zapier-d23d8967:

⠏ Installing project dependenciesadded 21 packages in 1.176s
✔ Installing project dependencies
✔ Applying entry point file
✔ Building app definition.json
⠙ Validating project>> GET https://zapier.com/api/platform/cli/apps/9298
<< 200
<< {"latest_core_version": "7.6.1", "image": null, "id": 9298, "invite_url": "https://zapier.com/platform/public-invite/9298/<stripped>/", "stats": {"Auth": {"Accounts": {"Paused": 4, "Live": 0, "Total": 4}, "Users": {"Paused": 1, "Live": 0, "Total": 1}}, "Trigger": {"Monitor Went Down": {"Paused": 2, "Live": 0, "Total": 2}, "Monitor Back Online": {"Paused": 0, "Live": 0, "Total": 0}}, "Action": {}, "Search": {}, "App": {"Totals": {"Paused": 7, "Live": 0, "Total": 7}}}, "all_versions": ["1.0.0"], "core_versions": ["7.6.1"], "intention": null, "role": null, "public": false, "pending": false, "app_category": null, "description": null, "key": "App9298", "service_id": null, "date": "2019-02-07T00:19:18", "slug": null, "versions": ["1.0.0"], "app_category_other": null, "title": "<striped>", "latest_version": "1.0.0", "homepage_url": null, "latest_core_npm_version": "7.6.1", "is_beta": null}

⠧ Uploading version 1.0.0>> PUT https://zapier.com/api/platform/cli/apps/9298/versions/1.0.0
>> {"zip_file":"raw zip removed in logs","source_zip_file":"raw zip removed in logs"}
<< 200
<< {"platform_version": "7.6.1", "user_count": 0, "oauth_redirect_uri": "https://zapier.com/dashboard/auth/oauth/return/App9298CLIAPI/", "changelog": null, "core_npm_version": "7.6.1", "app_id": 9298, "selected_api": "App9298CLIAPI@1.0.0", "deprecation_date": null, "environment": {}, "etag": null, "definition_override": null, "version": "1.0.0", "deployment": "non-production", "date": "2019-02-07T00:19:53", "id": 109362}

✔ Uploading version 1.0.0

Build and upload complete! You should see it in your Zapier editor at https://zapier.com/app/editor now!
```

## Your Environment

* CLI Version: 
zapier-platform-cli/7.6.0
zapier-platform-core/7.6.1
node/v8.11.2

* Operating System and version: MacOS Mojave 10.14
* App id: 9298

**Comment by ecdeveloper** [comment]
After digging in and debugging for a while, turned out there was a validation error which was silently swallowed (!!!) and the script pretended all went well. Closing this ticket, but it sounds like there are some problems which have to be fixed with the script not reporting about error.

**Comment by xavdid** [comment]
Thanks for the report! This was fixed in 388 and released as CLI `7.6.1`. If you update (`npm i -g zapier-platform-cli`) you should be all set.
